### ISSUE TITLE
MC 1.12 BC 2.4.0 Atlas crafting with empty recipe

### ISSUE THREAD

**Issue by Knito58** [Nuchaz/BiblioCraft#472]
Go to a standard crafting table. 
Put one item in the crafting grid.
Take it out again so that the grid is empty.
Bazinga! There's an Atlas in the output slot which is freshly crafted out of thin air.
This atlas seems to be usable normally and does not crash the game.
Forge: Latest (2343 atm)

**Comment by Nuchaz** [comment]
opps, must have had me a derp there. I was testing a custom recipe for the enchanted atlas, was meaning to comment it out, but didn't realize it was doing that. I'll try to get this fixed real soon

**Comment by Knito58** [comment]
Oha. Enchanted. Didn't realize that. 
Will get some more of them before you close this issue successfully.
:)

**Comment by Nuchaz** [comment]
I fixed this in the 2.4.1 update. The current enchanted atlases your getting will work like regular atlases, but  probably won't actually have the enchantment. Or at least, it'll show the enchantment, but it won't have any effect because the enchantment id will be different. I used to have the enchantment ID hardcoded, but they finally got rid of hardcoded ID's for enchantments (used to cause conflicts with other mods) so the enchantment will likely now have a different ID then the one that is saved to your current enchanted atlas the recipe bug gave you.
